marketplace_builder/views/pages/tutorials/form-configurations/configuration-options.liquid

```diff
@@ -0,0 +1,193 @@
+      Email
+    </label>
+
+    <input
+      class="form-control"
+      name="{{ form_builder.fields.properties.email.name }}"
+      value="{{ form_builder.fields.properties.email.value}}"
+      id="form_email"
+      type="email"
+     >
+
+     {{ form_builder.fields.properties.email.validation.rules.presence.message }}
+  </div>
+
+  <button class="btn btn-primary">Save</button>
+{% endform %}
+```
+{% endraw %}
+
+Configuration part of the forms defines API endpoint where server listen to specific request and invoke all acctions defined in the form. Implementation part is just what is displayed as HTML to your user. It is optional because sometimes you will only want to configure the server and the request will come from other source like in the example from tutorial: [Building a Contact Form with Customization](/tutorials/customizations/building-contact-form-with-customization#step-2-define-contact-form-–-form-object). For more information about implementation part head to [Rendering Form](/tutorials/form-configurations/rendering-form) tutorial.
+
+
+## List of required form configuration options
+
+### name
+`name` is uniq identifier of the form. It's how you reference your forms in [include_form tag](/api-reference/liquid-tags#include_form). This field doesn't depend of the actual FormConfiguration file name, but it's a good proactice that the file name correspond with the value of name property.
+
+{% include 'alert/warning', content: 'MarketplaceKit currently do not raise an error when two form configuration have the same name - last synced form will be displayed' %}
+
+### resoruce
+`resource` is either the name of your CustomModelType or one of predefined PlatformOS clasess. For the full list of classes visit [REST API reference page](api-reference/rest-api).
+
+Examples:
+* [Minimal form example](/tutorials/form-configurations/introduction#minimal-form-example) to see the example of **dynamic resource** based on CustomModelType
+
+* [Creating a Sign Up Form](/tutorials/users/creating-sign-up-form#step-2-configure-forms) for the **predefined class resource** example.
+
+
+## List of optional form configuration options
+
+### api_call_notifications
+`api_call_notifications` is array of api call notifications names that will be executed after the form is successfully submitted.
+
+Example from [Creating an API Call Notification](/tutorials/notifications/creating-api-call-notification
+):
+```yml
+api_call_notifications:
+  - ping_example_com_on_user_sign_up
+```
+
+### async_callback_actions
+`async_callback_actions` accepts two parameters:
+* delay - Integer, number of minutes added to the moment of form submition before the code defined in `content` is executed.
+* content - String, Liquid Template code executed asynchronusly. There is no timeout set for processing this code so it is the right place for more time-consuming calculations.
+
+Example:
+{% raw %}
+```yml
+async_callback_actions:
+  delay: 1
```

Comment:
Would be nice to add `priority` which defined how much time there is to accomplish given task before it timeouts.
Also max_attempts is pretty nice feature.

Read more: https://documentation.platform-os.com/release-notes/18-OCTOBER-2018

Reply:
done